packages/story-editor/src/utils/getCropParams.ts

```diff
@@ -13,12 +13,16 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+/**
+ * External dependencies
+ */
+import type { MediaElement } from '@googleforcreators/elements';
 /**
  * Internal dependencies
  */
 import { isOffCanvas } from './isOffCanvas';
 
-export function getCropParams(selectedElement) {
+export function getCropParams(selectedElement: MediaElement) {
```

Comment:
Just noting that in #12763 I defined a new `CropParams` type that could be used here for the return value.